src/styles/components/_keyboard.scss

```diff
@@ -0,0 +1,74 @@
+.keyboard {
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    padding: 5px 0;
+    background: $green;
+    box-shadow: 0 0 50px rgba(0, 0, 0, 0.5);
+    transition: bottom 0.4s;
+}
+
+.hidden {
+    bottom: -100%;
+}
+
+.keyboard-keys {
+    text-align: center;
+
+    .key {
+        height: 45px;
+        width: 6%;
+        max-width: 90px;
+        margin: 3px;
+        border-radius: 4px;
+        border: none;
+        background: rgba(255, 255, 255, 0.2);
+        color: $white;
+        font-size: 2rem;
+        outline: none;
+        cursor: pointer;
+        display: inline-flex;
+        align-items: center;
+        justify-content: center;
+        vertical-align: top;
+        padding: 0;
+        position: relative;
+
+        &:active {
+            background: rgba(255, 255, 255, 0.12);                      
+        }        
+    }
+
+    .wide-key {
+        width: 12%;
+    }
+
+    .extra-wide-key {
+        width: 36%;
+        max-width: 500px;
+    }    
+
+    .dark-key {
+        background: rgba(0, 0, 0, 0.25);
+    } 
+} 
+
+.activatable-key {
+    &::after {
```

Comment:
When you don't have any additional styling for class it can be written just in 1 line